City-of-Helsinki/drupal-hdbt
# UHF-8489 Helfi link button plugin

# [UHF-8489](https://helsinkisolutionoffice.atlassian.net/browse/UHF-8489)


### ❗ Do not merge as this branch contains code that should not be merged until a major refactoring❗ 

## What was done


* Modified the svg to sprite tool to fit 'button v2' and added styles for the button

## How to install and test


See instruction from https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/588

## Other PRs


* https://github.com/City-of-Helsinki/drupal-hdbt-admin/pull/232
* https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/588


[UHF-8489]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-8489?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
